communication/src/TlsConnectionManager.cpp

```diff
@@ -341,6 +355,17 @@ void ConnectionManager::connect() {
   }
 }
 
+void ConnectionManager::handleConnectTimeout(NodeNum i, const std::error_code& error_code) {
+  asio::steady_timer& deadline_timer_ = connecting_.at(i).second;
+  if (deadline_timer_.expiry() <= asio::steady_timer::clock_type::now()) {
+    LOG_WARN(logger_, "Timeout during connecting to node : " << i);
+    connecting_.at(i).first.close();
+    status_->num_connecting = connecting_.size();
+  }
+  LOG_TRACE(logger_, "Deadline Timer end");
```

Comment:
I have put trace so that these don't get dumped by default, if someone wants extra debugging by making changing log level to trace they could use it.

Reply:
In our system, we use DEBUG level for this purpose.